pkg/storagecapabilities/storagecapabilities.go

```diff
@@ -147,9 +148,9 @@ var storageClassToProvisionerKeyMapper = map[string]func(sc *storagev1.StorageCl
 	"csi.trident.netapp.io": func(sc *storagev1.StorageClass) string {
 		//https://netapp-trident.readthedocs.io/en/stable-v20.04/kubernetes/concepts/objects.html#kubernetes-storageclass-objects
 		val := sc.Parameters["backendType"]
-		if val == "ontap-nas" {
+		if strings.HasPrefix(val, "ontap-nas") {
```

Comment:
perhaps we want to do check for contains? instread of prefix?

Reply:
Yeah I am hesitant to start supporting all the different ones, so kept this strictly about ontap-nas/ontap-san error we saw